configuration/exodus-storage.yaml

```diff
@@ -50,8 +50,6 @@ Resources:
         BlockPublicPolicy: true
         IgnorePublicAcls: true
         RestrictPublicBuckets: true
-      VersioningConfiguration:
```

Comment:
In the case where the buckets already exist, does this actually disable versioning there or does it just leave the current setting alone?  Removing something from a config management tool isn't always enough to reverse the effect of previous runs.

Reply:
Right, buckets created prior to this change won't have versioning disabled by the template and would need to do that manually.

Reply:
It worth checking if the s3_bucket Ansible module support it. If it does than we can think of way to execute it on existing bucket. WDYT?

Reply:
I don't think this something we need to worry about, as it won't be an issue going forward. I've already disabled versioning manually on exodus-cdn-dev/stage/prod buckets and developers can do what they like with their preexisting buckets.